web_src/js/components/RepoContributors.vue

```diff
@@ -93,6 +98,9 @@ export default {
   methods: {
     sortContributors() {
       const contributors = this.filterContributorWeeksByDateRange();
+      const min = dayjs(this.xAxisMin).format('YYYY-MM-DD');
+      const max = dayjs(this.xAxisMax).format('YYYY-MM-DD');
+      this.searchQuery = `${this.repoLink}/commits/branch/${this.repoBranch}/search?q=after:${min}, before:${max}, author:`;
```

Comment:
It needs to encode (escape) the query parameter correctly

Reply:
resolved in 5d9bf83667c08509f12f42c17f74154d2b631f7c
Added a method to compute the search query for each contributor